packages/gasket-typescript-tests/test/preset-api.spec.ts

```diff
@@ -1,5 +1,5 @@
 import type { Gasket, GasketConfigDefinition, Plugin } from '@gasket/engine';
-import '@gasket/preset-api';
+import * as api from '@gasket/preset-api';
```

Comment:
Why * as here?
```suggestion
import api from '@gasket/preset-api';
```

Same with other preset type tests